conmon-rs/common/proto/conmon.capnp

```diff
@@ -20,6 +20,8 @@ interface Conmon {
         bundlePath @1 :Text;
         terminal @2 :Bool;
         exitPaths @3 :List(Text);
+        logPath @4 :Text;
+        maxLogSize @5 :Int64;
```

Comment:
to start, we can drop maxLogSize. Instead, we can implement a ReopenLog request that takes a container. I think in CRI-O we'll deprecate log size tracking outside of the kubelet (though we'll need it for podman)